### ISSUE TITLE
Suggest developer to visit the project folder directly instead of the script folder

### ISSUE THREAD

**Issue by kingychiu** [LAION-AI/Open-Assistant#1161]
The onboarding guide https://github.com/LAION-AI/Open-Assistant/blob/main/CONTRIBUTING.md#developer-setup is great. However, it suggests these:

```
If you want to work on something, assign yourself to it or write a comment that you want to work on it and what you plan to do.

To get started with development, if you want to work on the backend, have a look at scripts/backend-development/README.md.
If you want to work on any frontend, have a look at scripts/frontend-development/README.md to make a backend available.
```

In fact, the documentation in the script folder is limited, and you can find the same and more information from the project folders directly.

Backend: https://github.com/LAION-AI/Open-Assistant/tree/main/backend
Frontend: https://github.com/LAION-AI/Open-Assistant/tree/main/website

**Comment by olliestanley** [comment]
Feel free to make a PR which changes those lines to point to the locations you mentioned!

**Comment by kingychiu** [comment]
Thanks, I created a PR, I hope this makes sense
https://github.com/LAION-AI/Open-Assistant/pull/1171

Also, I am not sure are these sections
 `REST Server Configuration` and 
`Running the REST Server locally for development`
 still needed in https://github.com/LAION-AI/Open-Assistant/tree/main/backend?

**Comment by olliestanley** [comment]
Closed by #1171
